src/interpret.cpp

```diff
@@ -574,10 +574,11 @@ unsigned invokeNativeSlow(Thread* t, GcMethod* method, void* function)
   uint64_t result;
 
   if (DebugRun) {
+    signed char *cname = method->class_() && method->class_()->name() ? method->class_()->name()->body().begin() : (signed char*) "?";
```

Comment:
We should be able to declare `cname` as a `const char*` and avoid the cast here and elsewhere.  Also, we prefer to avoid C-style casts, since they're too general (i.e. they don't communicate the intent of the programmer as well as the more specific C++ casts).  Better to use static_cast, const_cast, or reinterpret_cast depending on the goal.